app/components/shared/inputs/TwitchTagsInput.vue.ts

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import { Inject } from 'util/injector';
+import { Multiselect } from 'vue-multiselect';
+import { $t, I18nService } from 'services/i18n';
+import { prepareOptions, TwitchTag, TwitchTagWithLabel } from '../../../services/platforms/twitch/tags';
+
+@Component({
+  components: {
+    Multiselect,
+  },
+})
+export default class TwitchTagsInput extends Vue {
+  @Inject() i18nService: I18nService;
+
+  @Prop() value: TwitchTagWithLabel;
+
+  @Prop() tags: Array<TwitchTag>;
+
+  selectPlaceholder = $t('Select stream tags');
+
+  tagPlaceholder = $t('Tag not found');
+
+  get options() {
+    return prepareOptions(
+      this.i18nService.state.locale || this.i18nService.getFallbackLocale(),
+      this.tags,
+    );
+  }
+
+  onInput() {
+    this.$emit('input', this.value);
+  }
+}
```

Comment:
I believe `prepareOptions` and taking `tags` as prop make it kind of specific, but open to suggestions. Not sure if we'd be abstracting too early, specially since the UI state is not good at the moment.

Reply:
it could just be something like `TaggableInput`, i just bring it up because facebook and other platforms have tags we may want to implement as well

Reply:
I think this is still too coupled to Twitch. `prepareOptions` is Twitch specific, and so are the types used in the component. In addition, the way that tags work on Twitch (e.g you can't add a new one, thus that specific placeholder) is engraved in here. So is the fact that it doesn't have search (for now), while the other platforms might. I don't think in it's current state this would be reusable at all. But I'm definitely happy to generalize once we're confident with it (at this moment I am not).

Reply:
That is sad I can't reuse this component for other purposes but I have it in the `shared` folder. If it's a very specific component for `Twitch` can you split it into 2 components - `TagsInput` and `TwitchTagsInput`